static/src/stylesheets/module/commercial/paidfor/_paidfor-meta.scss

```diff
@@ -1,15 +1,12 @@
 .paidfor-meta {
-    @include f-textSans;
-    background: $labs-main;
+    @include fs-textSans(3);
```

Comment:
Some of these classes are also in the amp stylesheets:

https://github.com/guardian/frontend/blob/59e3f4df36230492783d4771a379cea8eba89607/common/app/views/fragments/amp/stylesheets/glabs.scala.html

Do they also need updating there? No worries if fixing paid for AMP is outside of this PR!